lib/common/utils.js

```diff
@@ -82,7 +82,17 @@ export const sequence = function sequence(tasks, fn) {
 }
 
 export const parallel = function parallel(tasks, fn) {
-  return Promise.all(tasks.map(fn))
+  if (process.platform === 'win32') {
```

Comment:
This will restrict the performance  windows devices which have enough calculation resources, and also parallel is not enabled in building by default, the reason may be something else?